### PR TITLE
Updated dataset viewer

### PR DESCRIPTION
**Pull request by gentlecat**
I made it similar to the editor, which is also based on [React](https://facebook.github.io/react/).

*Live at http://test.acousticbrainz.org/ ([example](http://test.acousticbrainz.org/datasets/53ac14a7-c870-4e61-99d6-50336bdf1f27)).*

### CODE REVIEW

File: templates/datasets/viewer.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
```
Comment (alastair) [inline templates/datasets/viewer.html]: why did you rename this? `view` sounds like a good name

File: static/js/dataset-editor.js [hunk 2]
```diff
@@ -281,7 +282,7 @@ var ClassList = React.createClass({
                 <h4>Classes</h4>
```
Comment (alastair) [inline static/js/dataset-editor.js]: Is this normal react style to add layout directly to code - or can it be extracted out into template files?

File: templates/datasets/viewer.html (hunk 1 above)
Comment (gentlecat) [inline templates/datasets/viewer.html]: Just wanted naming of templates to be consistent. We have *delete*, *editor*, and *view*. Perhaps I should rename *editor* instead. :confused:

File: static/js/dataset-editor.js (hunk 2 above)
Comment (gentlecat) [inline static/js/dataset-editor.js]: React is meant to be used for UI, so I'd say yes. Take a look at https://facebook.github.io/react/docs/thinking-in-react.html, for example.
